changelog.md

```diff
@@ -4,6 +4,7 @@
 
 ### Version 5.0.9
 
+* `Dockerform`: Added option to deploy external databases using docker compose task    
```

Comment:
This should be the name of the Gradle plugin. Can you change to `cordformation` please?